README.md

```diff
@@ -43,35 +44,6 @@ following:
 *   _clang-format 5.0.1+_ (ships as part of _clang_)
 
 
-### Mac OS
```

Comment:
If we remove the complete section __Mac OS__, I think we should add Mac to the __Linux__ section title (so something like: __Linux and Mac OS__ as the title) to show that it is a supported OS (or at least that we intend to support it).

If we do not support it officially, we should mention something similar to the __Windows__ case.

Reply:
You are right. I dropped this because everything is the same so far except for when the Mac User uses Apple LLVM. I  will add **and Mac OS** to the title.